Cargo.toml

```diff
@@ -144,12 +144,10 @@ rust.nonstandard_style = "deny"
 rust.rust_2018_idioms = "deny"
 rust.trivial_casts = "deny"
 rust.trivial_numeric_casts = "deny"
-rust.unconditional_recursion = "deny"
```

Comment:
Because of false-positives, right? Maybe add a todo? This lint seems useful

Reply:
they are reported as warnings

Reply:
would you like me to return it to `deny`?

Reply:
Warnings are ok